src/js/components/DataFilters/DataFilters.js

```diff
@@ -66,10 +66,10 @@ export const DataFilters = ({ drop, children, heading, layer, ...rest }) => {
   let content = children;
   if (Children.count(children) === 0) {
     let filtersFor;
-    if (!properties && data && data.length)
+    if (!properties && unfilteredData && unfilteredData.length)
       // build from a piece of data, ignore object values
-      filtersFor = Object.keys(data[0]).filter(
-        (k) => typeof data[0][k] !== 'object',
+      filtersFor = Object.keys(unfilteredData[0]).filter(
```

Comment:
Is it reasonable for us to only be looking at the first object to build up the default options? This assumes the first object is representative of all objects in the data set and has all properties. What if other objects have properties not represented?

Reply:
I think it's reasonable given they can specify properties if they have sparse objects for some reason